app/forms/access_form.rb

```diff
@@ -1,15 +1,26 @@
 # frozen_string_literal: true
 
 # A form object for updating the rights on an item
+# @abstract
 class AccessForm
   extend ActiveModel::Naming
-
-  # @param [Cocina::Models::DRO]
-  def initialize(model, apo)
+  # @param [Cocina::Models::DRO, Cocina::Models::Collection] model
```

Comment:
Can you add `@param` documentation for new `default_rights` param?

Reply:
adding param documentation here